src/graph/GoExecutor.cpp

```diff
@@ -453,6 +455,15 @@ void GoExecutor::stepOut() {
                            << "error code: " << static_cast<int>(error.second);
             }
         }
+        if (FLAGS_trace_go) {
+            LOG(INFO) << "Step:" << curStep_
+                      << " finished, total request vertices " << starts_.size();
+            auto& hostLatency = result.hostLatency();
+            for (size_t i = 0; i < hostLatency.size(); i++) {
+                LOG(INFO) << hostLatency[i].first << ", time cost " << hostLatency[i].second
```

Comment:
Is it possible to launch multiple storaged on one host？Would it be better to print the port number as well？

Reply:
HostAddr includes ip and port.